test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +60,14 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
No need to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.